### PR TITLE
update content, formatting

### PR DESCRIPTION
**Pull request by tst-jcupp**

### CODE REVIEW

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 1]
```diff
@@ -195,34 +170,31 @@ Portable Globe and Server 5.2
...
-      .. rubric:: Drawing the polygon
+      .. rubric:: Drawing the Polygon
 
-      Once the globe name has been specified, you can define the
-      geographic region to be cut by drawing a polygon on the globe.
-      There are two ways to draw the polygon, one of which has been
-      deprecated.
+      Once the portable name has been specified, you can define the
+      geographic region to be cut by drawing a polygon. There are two ways to
+      draw the polygon, one of which has been deprecated.
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: If this is the same as the Spartan viewer Cutter updates, pull in updates from this PR
https://github.com/google/earthenterprise/pull/1614

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 2]
```diff
@@ -255,25 +227,24 @@ Portable Globe and Server 5.2
...
 
-      The polygon you specified in the previous step defines your 'area of
-      interest.' This area will contain high-resolution imagery and
-      data, and search tabs will be created for information that lies
-      within this zone. The maximum and minimum resolutions are
-      specified as integers between 1 and 24. These correspond to the
-      zoom levels that are used in the Fusion server.
+      The polygon made in the previous step defines your *area of
+      interest*. This area will contain high-resolution content, and search
+      tabs will be created for information that lying within. The maximum and
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: lies within it

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 3]
```diff
@@ -255,25 +227,24 @@ Portable Globe and Server 5.2
...
+      minimum resolutions are specified as whole numbers between 1 and 24. These
+      correspond to the zoom levels used in GEE Fusion.
 
-      .. rubric:: Minimum resolution
+      .. rubric:: Minimum Resolution
 
       The zoom level for the polygon area is specified in the **Maximum
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: Again, if this is now the Spartan view Cutter, I think this is the world level resolution?

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 4]
```diff
@@ -384,32 +356,38 @@ Portable Globe and Server 5.2
...
 
+
+
+
+
+
+
+
       .. _Portable_Server:
       .. rubric:: Portable server
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: I think we've been capping titles, so Portable Server

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 5]
```diff
@@ -384,32 +356,38 @@ Portable Globe and Server 5.2
...
 
          ``python portable_server.py``
 
       #. Navigate to the Portable Server administrative page at
          ``http://localhost:9335``. If you have set a custom ``port``
-         number in ``portable.cfg``, substitute that for "9335".
+         number in ``portable.cfg``, substitute that for ``9335``.
 
-      You can select what globe is being served from the administrative
+      You can select what portable is being served from the administrative
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: which portable

File: docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst [hunk 6]
```diff
@@ -384,32 +356,38 @@ Portable Globe and Server 5.2
...
       page. Click the |Portable folder icon| folder icon and you
       will see a screen where you can select what globe or map
```
Comment (tst-swanson) [inline docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst]: which globe

**Comment by tst-ccamp** [comment]
@tst-jcupp this is still marked as draft. Are you still wanting this to go into 5.3.3?

**Comment by tst-ccamp** [comment]
I haven't seen any progress on this in a while, so I'm closing it. You can reopen if you want

**Pull request closed without merging**